### ISSUE TITLE
Trying to login with empty fields results in a 500 error

### ISSUE THREAD

**Issue by furiml** [shinken-monitoring/mod-webui#547]
Hello,

While showing WebUI to one of my colleague, he clicked on the login button without writing anything in the login fields, and it resulted in a 500 error. He was using IE, and the login page is somehow very distorted, plus there was no "username" and "password" grey text : so, to quote him,  

> it appears, those 2 big blank rectangles are the form fields!

"Yup. But you shouldn't use IE."

Here's the full traceback : 

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 1732, in wrapper
    rv = callback(*a, **ka)
  File "/var/lib/shinken/modules/webui2/module.py", line 521, in lock_version
    return f(**args)
  File "/var/lib/shinken/modules/webui2/plugins/login/login.py", line 117, in user_auth
    is_authenticated = app.check_authentication(login, password)
  File "/var/lib/shinken/modules/webui2/module.py", line 811, in check_authentication
    user = self.auth_module.check_auth(username, password)
  File "/var/lib/shinken/modules/webui2/submodules/auth.py", line 95, in check_auth
    if self.check_cfg_password_auth(username, password):
  File "/var/lib/shinken/modules/webui2/submodules/auth.py", line 158, in check_cfg_password_auth
    p = c.password
AttributeError: 'Contacts' object has no attribute 'password'
```

After further testing, it only happens when the username is empty.

He suggested that when the fields are empty, the login button should be greyed out, and I think this is  a nice idea !

Furiml

**Comment by mohierf** [comment]
Thanks for reporting

**Comment by mohierf** [comment]
Closed thanks to #612
